packages/sanity/src/core/releases/store/createReleaseOperationStore.ts

```diff
@@ -159,6 +167,35 @@ export function createReleaseOperationsStore(options: {
       },
     ])
 
+  const handleRevertRelease = async (
+    revertReleaseId: string,
+    releaseDocuments: RevertDocument[],
+    releaseMetadata: ReleaseDocument['metadata'],
+    revertType: 'staged' | 'immediately',
+  ) => {
+    await handleCreateRelease({
+      _id: revertReleaseId,
+      metadata: {
+        title: releaseMetadata.title,
+        description: releaseMetadata.description,
+        releaseType: 'asap',
+      },
+    })
+    await Promise.allSettled(
```

Comment:
Nice use of `allSettled`! I do wonder if we want to surface it somehow if any of these fails, though. Also think there should be some concurrency management here we avoid hundreds of simultaneous requests. Although this might be just a stop-gap implementation for now? If so, then all good.